knex/migrations/20200512231206_fix-writer-education-id-column.js

```diff
@@ -1,11 +0,0 @@
-exports.up = function(knex) {
-  return knex.schema.table('writer_educations', tbl => {
-    tbl.increments();
-  })
-};
-
-exports.down = function(knex) {
-  return knex.schema.table('writer_educations', tbl => {
-    tbl.dropColumn('id');
-  })
-};
```

Comment:
Thanks for trying to tidy up! Unfortunately, we need this duplicate migration because it was run on the database once and so it will now look for this file every time we run migrations. This is here because there was an error in the first version of it when it was run on Heroku. Luckily though, the id isn't duplicated in the database so there's no duplicates in the actual db.